cli/productutil.py

```diff
@@ -82,6 +82,30 @@ def parse_dependency_str(token):
     return parse_dependency_str
 
 
+def rescan_product_images(
+    protecode_cfg_name: str,
+    product_cfg_file: CliHints.existing_file(),
```

Comment:
btw (just an idea that just came to my mind): how about a CliHints.yaml_file that would already do the yaml-parsing?